src/components/accordion/accordion.jsx

```diff
@@ -1,33 +1,35 @@
-var classNames = require('classnames');
-var React = require('react');
+const bindAll = require('lodash.bindall');
```

Comment:
@rschamp should www be using `import` instead of `require` to be consistent with other scratch-* projects?

Reply:
I think we should, but maybe in a separate PR, since we will have to update all the exports too, deal with `default` vs named exports, etc.  Could get hairy.  Also may be a good thing to do at the time that we implement the l10n string extraction system which requires `import`?